### DISCUSSION TITLE
Can you provide a wrapper class for the conversion between Servlet and Armeria HttpRequest and HttpResponse?

### DISCUSSION THREAD

**Discussion by i1619khz** [line/armeria#5237] [Q&A] [answered]
I want to fully replace Spring MVC with Armeria, but there are several frameworks that rely on Servlet for tasks like retrieving HTTP parameters.

**Comment by jrhee17** [comment, marked as answer]
> there are several frameworks that rely on Servlet for tasks like retrieving HTTP parameters.

Hi, Armeria provides an API to retrieve parameters.

```java
Server.builder().service("/", (ctx, req) -> {
    QueryParams queryParams = ctx.queryParams();
    ..
});
```

> wrapper class for the conversion between Servlet and Armeria HttpRequest and HttpResponse?

It might help if you can explain in more detail why you would like this
